### DISCUSSION TITLE
how to change target file to open when 'npm run docs-serve' is run?

### DISCUSSION THREAD

**Discussion by ebiya3** [twbs/bootstrap#39263] [Q&A]
I downloaded Bootstrap with npm, set it up and ran ''npm run docs-serve' and yes voilà.  All was well. So I set to work building a site and created a 'Dist' file with all my new pages. Now I want to open my 'Dist' file but it opens the bootstrap file. 
How do I change the path to open my 'Dist'  file.

**Comment by JamesVanWaza** [comment]
@ebiya3 post, your `package.json` file

**Reply by ebiya3** [reply]
@JamesVanWaza Thank you for the reply. 

**Edited by maintainer: removed file link for security reasons**

**Reply by julien-deramond** [reply]
I've edited your comment @ebiya3 
Please try to avoid adding files for security reasons. In order to share your `package.json`, you can copy and paste it here by using [Quoting code in Markdwon](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code)
